app/controllers/cloud_object_store_container_controller.rb

```diff
@@ -12,6 +12,11 @@ def breadcrumb_name(_model)
     ui_lookup(:tables => "cloud_object_store_container")
   end
 
+  def display_cloud_object_store_objects
+    @center_toolbar = 'cloud_object_store_objects'
```

Comment:
@miha-plesko i am little confused, about this why is center_toolbar being set here in controller. We have method in https://github.com/ManageIQ/manageiq-ui-classic/blob/master/app/helpers/application_helper/toolbar_chooser.rb#L5 that sets the toolbar to that should be loaded on the screen, are you bypassing that method and setting toolbar name in controller?